pf_driver/include/pf_driver/pf/pf_interface.h

```diff
@@ -37,48 +39,50 @@ class PFInterface
   void terminate();
 
 private:
-  using PipelinePtr = std::unique_ptr<Pipeline<PFPacket>>;
 
-  ros::NodeHandle nh_;
-  std::string ip_, port_;
-  ros::Timer watchdog_timer_;
-  std::unique_ptr<Transport> transport_;
-  transport_type transport_type_;
-  std::shared_ptr<PFSDPBase> protocol_interface_;
-  std::unique_ptr<dynamic_reconfigure::Server<pf_driver::PFDriverR2000Config>> param_server_R2000_;
-  std::unique_ptr<dynamic_reconfigure::Server<pf_driver::PFDriverR2300Config>> param_server_R2300_;
+    using PipelinePtr = std::unique_ptr<Pipeline<PFPacket>>;
 
-  enum class PFState
-  {
-    UNINIT,
-    INIT,
-    RUNNING,
-    SHUTDOWN,
-    ERROR
-  };
-  PFState state_;
-  std::string product_;
-  std::string expected_device_;
+    ros::NodeHandle nh_;
+    std::string ip_, port_;
+    std::string start_angle_, max_num_points_scan_;
+    ros::Timer watchdog_timer_;
+    std::unique_ptr<Transport> transport_;
+    transport_type transport_type_;
+    std::shared_ptr<PFSDPBase> protocol_interface_;
+    std::unique_ptr<dynamic_reconfigure::Server<pf_driver::PFDriverR2000Config> > param_server_R2000_;
+    std::unique_ptr<dynamic_reconfigure::Server<pf_driver::PFDriverR2300Config> > param_server_R2300_;
+
+    enum class PFState
+    {
+        UNINIT,
+        INIT,
+        RUNNING,
+        SHUTDOWN,
+        ERROR
+    };
+    PFState state_;
+    std::string product_;
+    std::string expected_device_;
 
-  HandleInfo info_;
-  ScanConfig config_;
-  ScanParameters params_;
+    HandleInfo info_;
+    ScanConfig config_;
+    ScanParameters params_;
```

Comment:
Please take care with the formatting. Whitespaces should not be introduced unnecessarily.